.github/workflows/translations.yml

```diff
@@ -41,7 +41,8 @@ jobs:
           # files when a new translation string is added but we don't need to
           # commit those until folks actually translate the new strings.
           if git diff translations | grep -qE '^[-+]msgstr ".+"$'; then
-            git add translations
+            make electron/src/renderer/assets/translations.json
```

Comment:
I think this won't work unless we do `yarn install` in the electron directory.  Otherwise gettext-compile won't work.

We could also see if we can just put the gettext-compile program in bin/, but it may have dependencies on other node modules.

Reply:
I believe we call yarn in messages.po in the makefile. make messages.po is called earlier in translations.yml. So I assumed it should be working.